composer/loggers/file_logger.py

```diff
@@ -27,23 +53,28 @@ class FileLogger(LoggerCallback):
 
     Args:
         filename (str, optional): File to log to.
-            Can be a filepath, ``stdout``, or ``stderr``. (default: ``stdout``)
+            Can be a filepath, ``stdout``, or ``stderr``. Default: ``stdout``.
         buffer_size (int, optional): Buffer size. See :py:func:`open`.
-            (default: ``1`` for line buffering)
-        log_level (LogLevel, optional): Maximum
-            :class:`~composer.core.logging.logger.LogLevel`. to record.
-            (default: :attr:`~composer.core.logging.logger.LogLevel.EPOCH`)
+            Default: ``1`` for line buffering.
+        log_level (LogLevel, optional):
+            :class:`~composer.core.logging.logger.LogLevel` (i.e. unit of resolution) at
+            which to record. Default:
+            :attr:`~composer.core.logging.logger.LogLevel.EPOCH`.
```

Comment:
Is there anyway to shorten the `:attr:`~composer.core.logging.logger.LogLevel.EPOCH`` throughout these docstrings? They make the docstrings impossible to read from the code itself (which is part of its purpose). If not, I'd almost prefer that they aren't linked, and we just provide a `see also` at the end linking to the events.

Reply:
solved, s/o @bandish-shah